src/redux/actions/rewards.js

```diff
@@ -67,13 +63,11 @@ export function doClaimRewardType(rewardType, options = {}) {
           reward: successReward,
         },
       });
-      if (successReward.reward_type === rewards.TYPE_NEW_USER) {
-        const action = doNotify({
-          message: 'You just earned your first reward!',
```

Comment:
@akinwale Not sure if this was a snackbar at one point, but the `displayType` isn't here. Should this use `doToast`?